### PR TITLE
Missing translations

### LINKED ISSUE #998: [Bug]: Not all strings on the site are translated to Spanish
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

We have lots of strings in the codebase we could be translating. E.g. in our filters and scholarship cards pretty much everything expect scholarship titles and descriptions and the the names of majors and schools are translatable.

### Steps To Reproduce

1. Go to https://dreamscholars.org/scholarships/x1dQqA69FKyz2qzwK9E1
2. Notice how words like "Apply", "Share", "Eligibility Requirements", "State", "Grades", "Schools", "Any", etc. aren't translated.

### What devices are you experiencing the problem on?

Desktop

### What browsers are you seeing the problem in?

Chrome

### PR DESCRIPTION
**Pull request by edwingl10**
## Motivation and Context

Fixes #998 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] User visible change (users will notice UI or functional changes)

## How Has This Been Tested?

- [ ] Existing or new tests cover my changes.
- [ ] Manually tested locally or on the Render PR Server.

## Previewing Changes

1. Click the `onrender.com` URL the **render `bot`** posted below.
2. go to 'add scholarships' 
3. change language

### CODE REVIEW

**Review by josueetcom — APPROVED** [review]

File: public/locales/en/scholarshipForm.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "generalDescription": "Please add the general information about the scholarship.",
+  "scholarshipName": "Scholarship Name",
+  "organization": "Organization",
+  "scholarshipLink": "Scholarship Link",
+  "deadline": "Deadline",
+  "description": "Description",
+  "tags": "Tags",
+  "awardAmount": "Award Amount",
+  "requirementsDescription": "Include information that is required for applicants to have.",
+  "noEligibilityReqs": "NO ELIGIBILITY REQUIREMENTS",
```
Comment (josueetcom) [inline public/locales/en/scholarshipForm.json:11]: This doesn't need to be in all caps btw. We can always do `t(...).toUpperCase()`.

File: public/locales/en/scholarshipForm.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "gradesPlaceholder": "No grade requirements",
+  "minGpa": "Minimum GPA",
+  "schools": "School(s)",
+  "schoolsPlaceholder": "No school requirements",
+  "states": "State(s)",
+  "statesPlaceholder": "No state requirements",
+  "majors": "Major(s)",
+  "majorsPlaceholder": "No major requirements",
+  "ethnicity": "Ethnicity(s)",
+  "ethnicityPlaceholder": "No ethnicity requirements",
```
Comment (josueetcom) [inline public/locales/en/scholarshipForm.json:22]: Maybe for the empty placeholders we have a common string like "no requirements"?

File: public/locales/es/common.json [hunk 3]
```diff
@@ -25,16 +26,30 @@
...
     "subscribeForUpdates": "Suscríbete para actualizaciones",
     "reachOut": "Contacta con nosotros"
   },
 
   "actions": {
-    "signIn": "Iniciar Sesión"
+    "signIn": "Iniciar Sesión",
+    "apply": "Aplicar",
+    "share": "Compartir",
+    "back": "Atras",
```
Comment (josueetcom) [inline public/locales/es/common.json:38]:
```suggestion
    "back": "Atrás",
```

File: public/locales/es/common.json [hunk 4]
```diff
@@ -25,16 +26,30 @@
...
+    "copyLink": "Copiar enlace",
+    "reportIssue": "Reportar un problem"
   },
 
   "dashboard": "Tablero de control",
-  "welcome": "Bienvenido"
+  "welcome": "Bienvenido",
+  "currentlySharing": "Actualmente compartiendo",
+  "general": "General",
+  "eligibilityReqs": "Requisitos de Elegibilidad",
```
Comment (josueetcom) [inline public/locales/es/common.json:49]: Spanish doesn't generally capitalize words in titles the way English does :P
```suggestion
  "eligibilityReqs": "Requisitos de elegibilidad",
```

File: public/locales/es/common.json [hunk 5]
```diff
@@ -25,16 +26,30 @@
...
+  "review": "Revisión",
+  "none": "Ninguno",
+  "more": "mas",
+  "any": "Todos",
+  "moreScholarships": "Más Becas"
```
Comment (josueetcom) [inline public/locales/es/common.json:54]:
```suggestion
  "moreScholarships": "Más becas"
```

**Review by josueetcom — DISMISSED** [review]

**Review by josueetcom — APPROVED** [review]

File: public/locales/en/common.json [hunk 6]
```diff
@@ -8,6 +8,7 @@
     "browseScholarships": "Browse Scholarships",
     "addScholarship": "Add a Scholarship",
```
[the remark this answers is not in the document]
Comment (edwingl10) [reply public/locales/en/common.json]: removed from footer and left in actions

**Pull request merged**